Assets/Scripts/VoicelineManager.cs

```diff
@@ -65,11 +66,15 @@ public class VoicelineManager : MonoBehaviour
     
     public static AudioClip[] voicelines;
 
+    public static string[] PersistentVoicelineNames { get; private set; }
```

Comment:
Let's use an enum for this. And then have a property on the enum for "IsPersistent" or something.

Reply:
This is only here so that the bemis audio source can know whether the clip it's currently playing should persist between level restarts. Can't use an enum for that because AudioSource.clip is an AudioClip, whose only identifying info is AudioClip.name, which is a string.

Reply:
I left the comment in the wrong spot, but I meant let's refer to the clips as an Enumerated Integer (value being index like it is currently used), not a string based on file name.